code/game/area/space_station_13_areas.dm

```diff
@@ -38,11 +38,12 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	sound_environment = SOUND_AREA_SPACE
 
 /area/space/nearstation
+	name = "Nearstation"
```

Comment:
this area is used a lot, this is a bad name for something you'll see in the game

Reply:
I think I spoke a bit about it in Coderbus and it _shouldn't_ affect anything? I don't know when a player would/should see this area, or build an APC in it since it's always a space area (no electronics/atmos). If I'm wrong, I'll probably get rid of the name and tell Timber to make sure that doesn't catch on unit tests.